indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -76,9 +74,26 @@ def query(self, key: int, *args, **kwargs) -> Optional[bytes]:
 
         return result
 
-    def update(self, keys: Iterator[int], values: Iterator[bytes]):
-        raise NotImplemented()
-
-    def delete(self, keys: Iterator[int]):
-        raise NotImplemented()
-     
\ No newline at end of file
+    def update(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
+        """updates the keys if they exist
+        """
+        missed = []
+        for key in keys:
+            if len(self.query(key)) == 0:
+                missed.append(key)
+        if missed:
```

Comment:
did we decide we would raise exception if not found?

Reply:
We decided to raise an exception for now. If this will cause any problems later, we will change it.